xml_inscriptions/metadata_only/KY.Lou.SAM.L.1929.17.527.xml

```diff
@@ -107,8 +109,10 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>rma <gap reason="illegible" quantity="1" unit="character"/><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
typically, when you can't interpret letters in their context, they go in all-caps I think